code/modules/mob/living/living_defense.dm

```diff
@@ -65,6 +65,12 @@
 	//No armor? Damage as usual
 	if(armor_effectiveness == 0)
 		apply_damage(effective_damage, damagetype, def_zone, sharp, edge, used_weapon)
+		if(ishuman(src) && def_zone)
+			var/mob/living/carbon/human/H = src
+			var/obj/item/organ/external/o = H.get_organ(def_zone)
+			if (o && o.status & ORGAN_SPLINTED)
```

Comment:
Any damage and the splints shatter? No deal, add at least some degree of threshold. Slapping someone on the ass and watching their splints fucking annihilate is hilarious, but too stupid.